Platon/backend/tests/search_engine_test.py

```diff
@@ -264,3 +265,78 @@ def test_institution_search(self):
         self.assertEqual(200,actual_response.status_code)
         self.assertEqual(expected_result,json.loads(actual_response.data))
 
+
+class UpcomingEventsSearchTest(BaseTest):
```

Comment:
I think you forgot to include UpcomingEventsSearchTest into test.py folder.

Reply:
When I tried to run the your test, I got the following error:

FAIL: test_deadline_search (tests.search_engine_test.UpcomingEventsSearchTest)
Traceback (most recent call last):
  File "/Users/cakmadam98/Desktop/4.1/CmpE451/bounswe2020group7/Platon/backend/tests/search_engine_test.py", line 342, in test_deadline_search
    self.assertEqual(expected_result, json.loads(actual_response.data))
AssertionError: {'num[42 chars]m': 'ANIVAE 2021', 'date': 'Mar 28, 2021 - Mar[260 chars])'}]} != {'num[42 chars]m': 'ISC 2021', 'date': 'Jun 2, 2021 - Jun 4, [606 chars])'}]}
Diff is 1188 characters long. Set self.maxDiff to None to see it.